Source/Core/Common/Config/Enums.h

```diff
@@ -30,6 +30,7 @@ enum class System
   GFX,
   Logger,
   Debugger,
+  CemuHookUdpServer
```

Comment:
Please add the trailing comma so future changes have less noise in the diff.
```suggestion
  CemuHookUdpServer,
```